ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -643,10 +643,12 @@ def atan(self, name=None):
     def atanh(self, name=None):
         return ivy.atanh(self._ivy_array)
 
-    @with_unsupported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
     def std(self, axis=None, unbiased=True, keepdim=False, name=None):
         return paddle_frontend.Tensor(
-            ivy.std(self._ivy_array, axis=axis, keepdims=keepdim)
+            ivy.std(
+                self._ivy_array, axis=axis, correction=int(unbiased), keepdims=keepdim
+            )
```

Comment:
There is a `std` function in `paddle_frontend` now. We should use that here instead of `ivy.std`. The resulting code would look something like:
```python
def std(self, axis=None, unbiased=True, keepdim=False, name=None):
        return paddle_frontend.std(self, axis=axis, unbiased=unbiased, keepdim=keepdim)
```

Reply:
done , made the changes required i think now its ready to merge. 🚀 

thank you